package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "github-hooks-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
why is this dev?
this is not needed for the deployment?

Reply:
In the repo it is now under `dependencies`, but I noticed something else editing it back to `devDependencies`, but I have to find out what exactly.